configs/projects/pe-bolt-server-runtime-main.rb

```diff
@@ -11,9 +11,6 @@
   proj.setting(:openssl_version, '3.0')
 
   instance_eval File.read(File.join(File.dirname(__FILE__), '_shared-pe-bolt-server_with_ruby.rb'))
-
-  # TODO: Work around PE-36078 by using forked non-optimal solution
-  proj.component('rubygem-rubyntlm-fork')
```

Comment:
whoops! I should have removed this when I added the `proj.setting(:use_legacy_openssl_algos, true)` in the bolt server, sorry.

Reply:
ah, yeah i'll add it

Reply:
oh, lol, i hadn't noticed that Sean didn't include my update for `:use_legacy_openssl_algos`. Thanks, Cas!